### DISCUSSION TITLE
Pass own props in Custom Tooltip

### DISCUSSION THREAD

**Discussion by tusharK-compro** [gilbarbara/react-joyride#859] [Q&A] [answered]
is there any way to pass my own props in Custom tooltip

**Comment by gilbarbara** [comment]
Hey @tusharK-compro 
Not sure what you mean by that. Can you post a sandbox with your requirements?

**Reply by tusharK-compro** [reply]
I have a custom tooltip and there is a checkbox inside the tooltip , i want to update the state in my main react parent component by clicking on that checkbox

**Comment by gilbarbara** [comment, marked as answer]
You can render your component inside the tooltip, there's no need for a custom tooltip just for that.
